model/metric.go

```diff
@@ -185,7 +240,7 @@ func IsValidMetricName(n LabelValue) bool {
 		}
 		return utf8.ValidString(string(n))
 	default:
-		panic(fmt.Sprintf("Invalid name validation scheme requested: %d", NameValidationScheme))
+		panic(fmt.Sprintf("Invalid name validation scheme requested: %s", NameValidationScheme.String()))
```

Comment:
nit: since we implement `fmt.Stringer` anyways

```suggestion
		panic(fmt.Sprintf("Invalid name validation scheme requested: %v", NameValidationScheme))
```

Reply:
I don't understand the change though `%v` though, `%s` is more specific and thus better IMO.